doc/developer.texinfo

```diff
@@ -2446,34 +2436,99 @@ GNU Texinfo Manual (@uref{http://www.gnu.org/software/texinfo/manual/texinfo/tex
 
 @need 1000
 @item
-Build everything to be distributed and copy it to the temporary staging area
-@c not @uref{...} because we don't want robots to notice and crawl the link
-(http://plse.cs.washington.edu/daikon/staging-daikon/).
-Part of this step is to build a set of publication abstracts.  If there are
-any failures, then investigate and correct as necessary.
-The final output of this step
-will be a list of files that were added/removed since the
-last release.
-If any of these differences is unexpected, then investigate and correct the
-difference, and redo the step.
+Once you have successfully reached this point, you are ready to make a release
+candidate and begin testing it. Any source changes required to fix a problem
+encountered above must be committed to the repository now.
+Verify doc/CHANGES is up-to-date and there are no uncommitted files.
+
+@example
+make check-repo
+@end example
+
+@noindent
+The result of the command above should be:
```

Comment:
This doesn't say what to do in that case.  How about moving the text about "must be committed to the repository" to here, saying that if it doesn't say "nothing to commit" then you should commit the files.  This seems like a clearer place to write those instructions, since this is the place where the user will take action.